runtime-infra/celonis_exports.yaml

```diff
@@ -0,0 +1,736 @@
+      EnableKeyRotation: true
+      KeyPolicy:
+        Version: "2012-10-17"
+        Id: !Ref AWS::StackName
+        Statement:
+          - Sid: Allow data account to do everything
+            Effect: Allow
+            Principal:
+              AWS: !Sub arn:aws:iam::${AWS::AccountId}:root
+            Action:
+              - "kms:Create*"
+              - "kms:Describe*"
+              - "kms:Enable*"
+              - "kms:List*"
+              - "kms:Put*"
+              - "kms:Update*"
+              - "kms:Revoke*"
+              - "kms:Disable*"
+              - "kms:Get*"
+              - "kms:Delete*"
+              - "kms:ScheduleKeyDeletion"
+              - "kms:CancelKeyDeletion"
+              - kms:*Tag*
+              - kms:*tag*
+            Resource: "*"
+          - Sid: Allow target accounts to use key for encrypt/decrypt
+            Effect: Allow
+            Principal:
+              AWS:
+                - !Sub arn:aws:iam::${AWS::AccountId}:root
+            Action:
+              - kms:Encrypt
+              - kms:Decrypt
+              - kms:ReEncrypt*
+              - kms:GenerateDataKey*
+              - kms:DescribeKey
+            Resource: "*"
+
+
+
+###############################################################################
+###            USER FOR BUCKET ACCESS WITH SECRET CONTAINING KEYS           ###
+###############################################################################
+
+  CelonisReadOnlyUser:
+    Type: AWS::IAM::User
+
+  CelonisS3ReadOnlyPolicy:
+    Type: 'AWS::IAM::Policy'
+    Properties:
+      PolicyName: CelonisS3ReadOnlyPolicy
+      Users:
+        - !Ref CelonisReadOnlyUser
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Sid: ReadAndListBucketObjects
+            Effect: Allow
+            Action:
+              - "s3:GetBucketAcl"
+              - "s3:GetObject"
+              - "s3:ListBucket"
+            Resource:
+              - !Sub "arn:${AWS::Partition}:s3:::${CelonisHistoryBucket}"
+              - !Sub "arn:${AWS::Partition}:s3:::${CelonisHistoryBucket}/*"
+              - !Sub "arn:${AWS::Partition}:s3:::${CelonisCurrentDataBucket}"
+              - !Sub "arn:${AWS::Partition}:s3:::${CelonisCurrentDataBucket}/*"
+          - Sid: DecryptS3ObjectsAndSecret
+            Effect: Allow
+            Action:
+              - kms:Decrypt
+              - kms:DescribeKey
+            Resource:
+              - !Sub "${CelonisBucketsKmsKey.Arn}"
+          - Sid: GetSecret
+            Effect: Allow
+            Action:
+              - secretsmanager:GetSecretValue
+              - secretsmanager:DescribeSecret
+              - secretsmanager:ListSecretVersionIds
+              - secretsmanager:ListSecrets
+            Resource:
+              - !Ref UserKeysSecret
+          - Sid: ListSecrets
+            Effect: Allow
+            Action:
+              - secretsmanager:ListSecrets
+            Resource:
+              - "*"
```

Comment:
@marco-vit-pagopa possiamo restringere ai secets di questo account e questa region?

Reply:
Purtroppo sostituire l'asterisco con un espressione mi mandava in errore lo script che avevo preparato per celonis; ho cmbiato approccio.

Prima facevo la ricerca del giusto secret basandomi su un tag.

Invece adesso:
 - lo stack ha un output (e uno solo) che contiene l'arn del secret, 
 - consento al sistema esterno di effettuare la describe solo di quello stack.
 - lo script non cerca il secret ma ne fa una getValue diretta tramite ARN
 - lo script non deve cercare lo stack perché il nome è fisso